frontend/habragasolina-web/routes/web.php

```diff
@@ -5,3 +5,30 @@
 Route::get('/', function () {
     return view('welcome');
 });
+
+Route::get('/main', function () {
```

Comment:
Delete the route that is created originally and rename this to be the main page, so only the '/' is needed